ui-core/src/styles/button.css

```
@@ -28,6 +28,10 @@
     border-radius: 0.313rem;
   }
   &.primary {
+    &:focus {
+      outline: 0.125rem solid #256afa;
```

This file is a little outdate but a few months ago we decided to use pixels for all css except `font-size`, we can start to use it right now in this file (the rest can be done a in different PR).
Same for colors, we should use this syntax.

You can do the same for the rest of the css you changed in this PR :)
```suggestion
      outline: 2px solid theme('colors.primary.50');
```

---

Thanks for the feedback @SharglutDev .... I will work on this later this week when I have time :)